bokehjs/src/lib/models/text/utils.ts

```diff
@@ -0,0 +1,31 @@
+import {isString} from "core/util/types"
+import {TeX} from "models/text/index"
+import {BaseText} from "./base_text"
+
+export function is_tex_string(text: unknown): text is string {
```

Comment:
This doesn't make sense. You are asking if `text` is a TeX string, but you are asserting a `string`. This would make sense if you returned some more specific, TeX related, type. This function should take a string and return a simple boolean. Type checking should be done at usesite.